packetbeat/docs/flows.asciidoc

```diff
@@ -0,0 +1,104 @@
+[[flows]]
+== Configuring Flows to Monitor Network Traffic
+
+You can configure Packetbeat to collect and report statistics on network flows.
+A _flow_ is a group of packets sent over the same time period that share
+common properties, such as the same source and destination address and protocol.
+You can use this feature to analyze network traffic over specific protocols on
+your network.
+
+For each flow, Packetbeat reports the number of packets and the total number of
+bytes sent from the source to the destination. Each flow event also contains
+information about the source and destination hosts, such as their IP address and
+location. For bi-directional flows, Packetbeat reports statistics for the reverse
+flow.
+
+Packetbeat collects and reports statistics per flow on the following network
+layers:
+
+* IP/TCP
+* UDP
+* SSL
+
```

Comment:
Somewhere in here we should explain the `final` flag and how the packetbeat sends intermediate reports about each flow that it is tracking with the `final = false`, then when the flow completes it sends one last event with `final = true`. Users that wish to aggregate sums of traffic will want to know this because they should be filtering on `final:true` (or using some other technique to get only the latest update from each flow).